user_docs/en/userGuide.t2t

```diff
@@ -3659,13 +3659,13 @@ Due to this, and to maintain compatibility with other screen readers in Taiwan,
 
 ++ Eurobraille displays ++[Eurobraille]
 The b.book, b.note, Esys, Esytime and Iris displays from Eurobraille are supported by NVDA.  
-These devices have a braille keyboard with 10 keys. 
+These devices have a Braille keyboard with 10 keys. 

```

Comment:
The word "Braille" only needs to be capitalised when referring to the inventor of the reading system.  Referring to the code itself, lowercase is usual (as is done in the rest of this PR).  Ref: https://www.brailleauthority.org/capitalization/capitalization.html